auth-web/tests/unit/components/EntityManagement.spec.ts

```diff
@@ -1,15 +1,18 @@
-import { createLocalVue, mount } from '@vue/test-utils'
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import AddBusinessDialog from '@/components/auth/manage-business/AddBusinessDialog.vue'
+import AffiliatedEntityTable from '@/components/auth/manage-business/AffiliatedEntityTable.vue'
+import DashboardHelp from '@/components/auth/manage-business/DashboardHelp.vue'
 import EntityManagement from '@/components/auth/manage-business/EntityManagement.vue'
+import ModalDialog from '@/components/auth/common/ModalDialog.vue'
+import PasscodeResetOptionsModal from '@/components/auth/manage-business/PasscodeResetOptionsModal.vue'
 import { RemoveBusinessPayload } from '@/models/Organization'
 import Vue from 'vue'
-import VueI18n from 'vue-i18n'
 import VueRouter from 'vue-router'
 import Vuetify from 'vuetify'
 import Vuex from 'vuex'
 
 Vue.use(Vuetify)
 Vue.use(VueRouter)
-Vue.use(VueI18n)
```

Comment:
This might not be used in the unit tests, but it's likely used in the normal code for getting localized strings (re: [internationalization](https://kazupon.github.io/vue-i18n/)).